### ISSUE TITLE
[Feature Request] Selecting multiple manga to categorize in catalogue

### ISSUE THREAD

**Issue by vongdong** [Jays2Kings/tachiyomiJ2K#91]
Can you please implement my feature request? It would make bookmarking multiple manga from a catalogue so much faster than doing one at a time. Just like being able to select multiple manga from the categories to move or delete.

Would be great to select multiple manga to download at the same time too.

**Comment by joseph619** [comment]
I would like the first one too, current bookmarking method in Browse is pretty limiting. It would be greate if we could select multiple manga in Browse and bookmark them at the same time.

I don't think the downloads feature would be added though.

**Comment by CarlosEsco** [comment]
If you added this you would lose long press to quick add a manga.

**Comment by vongdong** [comment]
> If you added this you would lose long press to quick add a manga.

But what if I wanted to add multiple manga? I'd have to do them individually. Not sure about others but I read the manga description before adding.

If the multi select from the library could be implemented to the extensions then that would be great since it's only one extra step which is choosing which category to add them to.

**Comment by CarlosEsco** [comment]
That makes even less sense though. If you read the description why not just click the heart when there. 

But like I said if you long press and do multi select like the library you lose the ability to long press a manga and instantly add it to your library.

**Comment by vongdong** [comment]
> That makes even less sense though. If you read the description why not just click the heart when there.
> 
> But like I said if you long press and do multi select like the library you lose the ability to long press a manga and instantly add it to your library.

Well that still means I still have to add them individually. I could check out the ones I want to add then add them all together when I'm done checking them out.

Long press only works well if there's one set category for them to be added to.

**Comment by CarlosEsco** [comment]
Don't set a default category and then you are prompted on long press.  Im not seeing a benefit especially if you are reading descriptions

Below isn't even counting you clicking into a manga reading the description and not just clicking while in manga info to add the manga.

Current
X TIMES 3 (Long press, select category, ok)

Pros: can add individual manga, can pick categories specially for the manga
Cons: can't add multiple manga at once

Your proposed 
1+ X + 2
(Long press, click X number of chapters, click the categories, click ok)
Pros: less clicks to add multiple manga
Cons: more clicks to add one manga, can't choose unique categories per a manga

**Comment by vongdong** [comment]
> Don't set a default category and then you are prompted on long press. Im not seeing a benefit especially if you are reading descriptions
> 
> Below isn't even counting you clicking into a manga reading the description and not just clicking while in manga info to add the manga.
> 
> Current
> X TIMES 3 (Long press, select category, ok)
> 
> Pros: can add individual manga, can pick categories specially for the manga
> Cons: can't add multiple manga at once
> 
> Your proposed
> 1+ X + 2
> (Long press, click X number of chapters, click the categories, click ok)
> Pros: less clicks to add multiple manga
> Cons: more clicks to add one manga, can't choose unique categories per a manga

Yes, I believe I screwed my argument when I said I read descriptions before adding. What I actually meant is i read descriptions of manga that actually interest me so i do quickly add some manga without checking them out.

Okay for example:
There's 10 manga I want to add to my library into 3 separate categories, which is faster?
- multi selecting and adding to chosen categories
- selecting one at a time and adding to chosen category/categories

I don't really have a problem with the quick add, it would be better if I could choose a selected default category to add to faster instead of going through the menu to change it if you understand what i mean.

**Comment by joseph619** [comment]
This has been added with https://github.com/Jays2Kings/tachiyomiJ2K/commit/9e813f773b1a4265f3e40557623394ce65776757
